wiki_to_csv.py

```diff
@@ -123,7 +123,8 @@ def get_all_page_data(year):
     pages = cat.articles()
     data = OrderedDict()
     for page in pagegenerators.PreloadingGenerator(pages, 100):
-        if page.title().endswith("/Global Metrics"):
+        if (page.title().endswith("/Global Metrics") or
```

Comment:
Do we need or want support for the old template?

Reply:
We want to support the old _subpage name_ (since multi-year projects may still be using that name), we don't want to support the old _template name_ though since the parameters are different.